scripts/run_pier.sh

```diff
@@ -332,6 +332,25 @@ function pier_clean() {
     rm "${CURRENT_PATH}"/pier/pier-$MODE-binary.addr
   fi
 
+  if [[ ! -z `ps | grep ${CURRENT_PATH}/pier/.pier_$MODE/plugins/appchain_plugin | grep -v "grep"` ]]; then
+    echo "clean the plugin process for $MODE pier"
+    OLD_IFS=$IFS
+    IFS="
+"
+    list=`ps aux| grep ${CURRENT_PATH}/pier/.pier_$MODE/plugins/appchain_plugin | grep -v "grep"`
```

Comment:
ps aux | grep ${CURRENT_PATH}/pier/.pier_$MODE/plugins/appchain_plugin | grep -v "grep"`  |awk '{print $2}' to get pid is better?